curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/task-45.md

```diff
@@ -0,0 +1,60 @@
+---
+id: 657cb68bf15f349a744b5fba
+title: Task 45
+challengeType: 19
+dashedName: task-45
+---
+
+# --description--
+
+When you want to learn about a person’s character or personality without specifying their gender, you can use `they` as a singular pronoun and ask, `What are they like?`.
+
+This is a respectful and inclusive way to ask about someone. Using `they` in this way ensures that you don't make assumptions about people.
+
+Example:
+
+Imagine there is a new team member joining your programming project, but you haven’t met them yet. 
+
+Instead of assuming their gender, you can ask your supervisor, `What are they like?` 
+
+This question helps you find out about the new member's personality, such as being organized, creative, or detail-oriented.
+
+**Note:** The question `What are they like?` can also refer to multiple people or a group. For example, if you're asking about a new team or a group of people you will be working with, you can use this question to learn about the group's personality or dynamic.
```

Comment:
```suggestion
When you want to learn about a person’s character or personality without specifying their gender, you can use `they` as a singular pronoun and ask, `What are they like?`.

This is a respectful and inclusive way to ask about someone. Using `they` in this way ensures that you don't make assumptions about people. Example:

Imagine there is a new team member joining your programming project, but you haven’t met them yet. Instead of assuming their gender, you can ask your supervisor, `What are they like?` 

This question helps you find out about the new member's personality, such as being organized, creative, or detail-oriented.

**Note:** The question `What are they like?` can also refer to multiple people or a group. For example, if you're asking about a new team or a group of people you will be working with, you can use this question to learn about the group's personality or dynamic.
```